tests/dep_test_permutation.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Testing File
+"""
+# Import necessary modules
+import numpy as np
+import pyrft as pr
+import os
+
+# The location where the pyrft module is saved (need to specify yourself)
+pyrft_loc = 'C:/Users/12SDa/davenpor/davenpor/Toolboxes/pyrft/'
+
+# Where data is stored
+data_store = pyrft_loc + 'tests2/test_data/'

```

Comment:
Do you really need data for your tests ? 
Overall it is bad practice to hardcode paths in such a way. If you really need data that are part of pyrft, you should simply import them.